### PR TITLE
chore: update deps

### PR DESCRIPTION
**Pull request by psav**

### CODE REVIEW

File: go.mod [hunk 1]
```diff
@@ -115,14 +112,15 @@ require (
...
 	github.com/pmezard/go-difflib v1.0.0 // indirect
-	github.com/prometheus/client_model v0.4.0 // indirect
-	github.com/prometheus/common v0.42.0 // indirect
-	github.com/prometheus/procfs v0.9.0 // indirect
+	github.com/prometheus/client_model v0.5.0 // indirect
+	github.com/prometheus/common v0.45.0 // indirect
+	github.com/prometheus/procfs v0.12.0 // indirect
 	github.com/quic-go/qpack v0.4.0 // indirect
 	github.com/quic-go/qtls-go1-20 v0.3.1 // indirect
 	github.com/quic-go/quic-go v0.37.7 // indirect
```
Comment (Victoremepunto) [inline go.mod:120]: @psav  please can we bump this ? seems there's a vuln - ideally latest available (0.42.0)

**Review by Victoremepunto — CHANGES_REQUESTED** [review]

File: go.mod (hunk 1 above)
Comment (psav) [reply go.mod:120]: 0.4.0 -> 0.42.0 ????

Comment (psav) [reply go.mod:120]: or 0.4.2???

Comment (Victoremepunto) [reply go.mod:120]: this => https://github.com/quic-go/quic-go/releases/tag/v0.42.0

Comment (Victoremepunto) [reply go.mod:120]: check the Github Actions pipeline

**Review by Victoremepunto — APPROVED** [review]

**Pull request merged**